big_tests/tests/service_mongoose_system_metrics_SUITE.erl

```diff
@@ -126,6 +148,28 @@ init_per_testcase(module_backend_is_reported, Config) ->
     create_events_collection(),
     maybe_start_module(mim(), mod_vcard),
     enable_system_metrics(mim()),
+    Config;
+init_per_testcase(mongoose_version_is_reported, Config) ->
+    create_events_collection(),
+    enable_system_metrics(mim()),
+    Config;
+init_per_testcase(cluster_uptime_is_reported, Config) ->
+    create_events_collection(),
+    enable_system_metrics(mim()),
+    Config;
+init_per_testcase(xmpp_componenets_are_reported, Config) ->
+    create_events_collection(),
+    Config1 = get_components(common(Config), Config),
+    escalus:create_users(Config1, escalus:get_users([alice, bob])),
```

Comment:
I think there is no need to create the users. At least not for the test case checking number of connected XMPP components.